pkgs/applications/editors/idea/default.nix

```diff
@@ -340,12 +340,12 @@ in
 
   datagrip = buildDataGrip rec {
     name = "datagrip-${version}";
-    version = "2016.3.2";
+    version = "2016.3";
```

Comment:
@globin @Mic92 I've updated the commit message and corrected the datagrip version.  I've also corrected the hash for it as that seems to have regressed.

I've tested datagrip too.